### PR TITLE
PR for Fixed formatting and other small website fixes

### PR DESCRIPTION
**Pull request by mewmutt**
This is a PR for small website fixes, and other things that go for fixed code, formatting issues,  The way that I made the contact part, was so there could be an address at some point. This means fixing it to recenter it inside of the page would take a long time. I personally think it's fine right now. Requesting Elikem for reviewer.

### CODE REVIEW

**Comment by mewmutt** [comment]
@blaisedampier , I want you to go through and analyze these files and see how they work. I want you to see how I did it and cleaned it up, and made the formatting easier to read and more compact. It's also good practice!

File: index.html [hunk 1]
```diff
@@ -114,159 +92,132 @@ <h2>Custom software built for your everyday business needs.</h2>
...
         <h2>Services</h2>
       </div>
-
       <div class="row">
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch" data-aos="fade-up">
           <div class="icon-box">
-            <div class="icon"><i class="bx bxl-dribbble"></i></div>
+            <div class="icon"><i class="bi-code-square"></i></div>
             <h4><a href="">Application Development & Mobile Application Development</a></h4>
             <p>We have a strong understanding of programming languages and the ability to solve complex problems. We also are able to work well in a team and communicate effectively with others. Additionally, we have deep understanding of the latest technologies and trends in the field</p>
```
Comment (AndrewEA95) [inline index.html]:
```suggestion
            <p>We have a strong understanding of programming languages and the ability to solve complex problems. We also are able to work well in a team and communicate effectively with others. Additionally, we have deep understanding of the latest technologies and trends in the field.</p>

```

This sentence is just missing a period!

File: index.html [hunk 2]
```diff
@@ -114,159 +92,132 @@ <h2>Custom software built for your everyday business needs.</h2>
...
           </div>
         </div>
-
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-md-0" data-aos="fade-up" data-aos-delay="150">
           <div class="icon-box">
-            <div class="icon"><i class="bx bx-file"></i></div>
+            <div class="icon"><i class="bi-file-earmark-binary"></i></div>
             <h4><a href="">System Automation</a></h4>
             <p>Authorative use of software, hardware and other tools to automate the administration, management, monitoring, and other operational tasks of computing. We have a strong understanding of a variety of programming languages and experience with automation tools and frameworks.</p>
```
Comment (AndrewEA95) [inline index.html]:
```suggestion
            <p> Authoritative use of software, hardware and other tools to automate the administration, management, monitoring, and other operational tasks of computing. We have a strong understanding of various programming languages and experience with automation tools and frameworks.</p>

```
Authoritative isn't spelled right.

File: index.html [hunk 3]
```diff
@@ -114,159 +92,132 @@ <h2>Custom software built for your everyday business needs.</h2>
...
           </div>
         </div>
-
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0" data-aos="fade-up" data-aos-delay="300">
           <div class="icon-box">
-            <div class="icon"><i class="bx bx-tachometer"></i></div>
+            <div class="icon"><i class="bi-cloud-download"></i></div>
             <h4><a href="">Legacy Software Intregration Development</a></h4>
```
Comment (AndrewEA95) [inline index.html]:
```suggestion
            <h4><a href="">Legacy Software Integration Development</a></h4>

```
Integration isn't spelled right.

File: index.html [hunk 4]
```diff
@@ -114,159 +92,132 @@ <h2>Custom software built for your everyday business needs.</h2>
...
-        
+        </div>  
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4" data-aos="fade-up" data-aos-delay="750">
           <div class="icon-box">
-            <div class="icon"><i class="bx bx-arch"></i></div>
+            <div class="icon"><i class="bi-filetype-ai"></i></div>
             <h4><a href="">A.I. Modeling</a></h4>
-              <p>(coming soon)</p>
+              <p>(Coming Soon)</p>
               <p>We plan on devloping advanced A.I models to help business crunch customer analytics and internal System analytics. Using A.I modeling, we would help business make better decisions in terms of predicting and showcasing the outcome and impact of those choice.</p>
```
Comment (AndrewEA95) [inline index.html]:
```suggestion
              <p>We plan on developing advanced A.I models to help business crunch customer analytics and internal System analytics. Using A.I modeling, we would help business make better decisions in terms of predicting and showcasing the outcome and impact of those choice.</p>

```
Developing isn't spelled right.

**Review by AndrewEA95 — COMMENTED** [review]
Good work on the changes Myles, Overall the PR looks good just a few typos is all I saw that needed to be changed!

File: index.html (hunk 2 above)
Comment (AndrewEA95) [reply index.html]: lol also if you want you can ignore the change from "a variety of " to "various" it makes no difference!

**Pull request merged**